Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -4,14 +4,14 @@ import PropTypes from 'prop-types';
 const NumberButton = ({
   symbol, style, onClick, action,
 }) => {
-  const clickListener = () => {
-    onClick(symbol);
-  };
-
   const setActionClass = () => (action ? 'action-button' : '');
 
   return (
-    <div onClick={clickListener} className={`number-button ${setActionClass()}`} style={style}>
+    <div
+      onClick={() => onClick(symbol)}
+      className={`number-button ${setActionClass()}`}
+      style={style}
+    >
       {symbol}
     </div>
```

Comment:
**Day 2 Review**

- Great job working on Calculator project Jakub You managed to implement display functionality , the numbers pressed are showing on the display  👍
- Next steps i would take is to try and make it so operators will work. Switch case in JS might be handy for this.
- Your code is clear and i like how You are using .map() generate new components and pass props to them. This is how You create reusable components.
Overall great job 👍